pkg/didmethod/httpbinding/resolver.go

```diff
@@ -0,0 +1,121 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package httpbinding
+
+import (
+	"crypto/tls"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"path"
+	"time"
+
+	"github.com/hyperledger/aries-framework-go/pkg/common/log"
+	errors "golang.org/x/xerrors"
+)
+
+var logger = log.New("aries-framework/didmethod/httpbinding")
+
+// optsOfHTTP holds options for the HTTP transport implementation of DIDResolver
+// it has a required http.Client instance and optional basePath which is appended to the remove DID Resolver URL
+type optsOfHTTP struct {
+	client *http.Client
+}
+
+// HTTPOpt is an outbound HTTP transport option
+type HTTPOpt func(opts *optsOfHTTP)
```

Comment:
HTTP  doesn't really seem needed - these are for the package constructor.

Reply:
Agree